src/app/rpmostree-builtin-upgrade.cxx

```diff
@@ -129,6 +167,48 @@ rpmostree_builtin_upgrade (int             argc,
         g_print ("note: automatic updates (%s) are enabled\n", policy);
     }
 
+  if (!opt_force)
+    {
+      g_autofree char *update_driver_sd_unit = NULL;
+      g_autofree char *update_driver_name = NULL;
+      if (!get_driver_info (&update_driver_name, &update_driver_sd_unit, error))
+        return FALSE;
+
+      /* Notify user that an updates driver is registered and upgrades should be
+       * done through the driver. */
+      if (update_driver_sd_unit && update_driver_name)
+        {
+          g_print ("This node is updated by %s", update_driver_name);
```

Comment:
"node" is a common term in Kubernetes, but not all computers are part of a cluster.  Maybe...let's just go for something simple like:
```
error: An update driver is registered: zincati.service; use --force to override
```
?

Or perhaps we shouldn't even have `--force`, instead tell people to use systemd like `systemctl stop zincati`?  Or as you're doing here, just point people at the documentation for the unit which should tell them how to do this?

I like the Documentation parsing, but...it might be a lot simpler to just invoke `systemctl status $unit` which nicely renders all this stuff to the terminal already.

Reply:
> instead tell people to use systemd like `systemctl stop zincati`?

Wouldn't this be an extra step though? Or are we intentionally making it harder so people think harder about their choices?

I thought actively showing the documentation made sense since it suggests to the user that they can do what they want to do through configurations to the updates driver instead. But yeah.. it would be a lot simpler to just show the systemd unit.

Reply:
Related question: in practice, the reason why people aren't on the newest release (the release we upgrade to when we do an `rpm-ostree upgrade`) is because:
- the updates driver is not checking for updates frequently enough (i.e. we're just waiting for e.g. Zincati to check if there's a release to update to)
- we're in the middle of a chain of barrier updates to get to the latest release, but due to some reboot strategy configuration, we're waiting in between the reboots (as opposed to the default immediate reboot strategy).
- We're at a deadend release.